source/entries.d.ts

```diff
@@ -0,0 +1,57 @@
+import {ArrayEntry, MapEntry, ObjectEntry, SetEntry} from './entry';
+
+type ArrayEntries<BaseType extends readonly unknown[]> = Array<ArrayEntry<BaseType>>;
+type MapEntries<BaseType> = Array<MapEntry<BaseType>>;
+type ObjectEntries<BaseType> = Array<ObjectEntry<BaseType>>;
+type SetEntries<BaseType extends Set<unknown>> = Array<SetEntry<BaseType>>;
+
+/**
+Many collections have an `.entries()` method that will return an iterable of that structure's keys and values. The `Entries` type will return the type of that collection's entries given the type itself.
```

Comment:
I'm not sure what "given the type itself" means. I think these two sentences could be more clearly written.

Reply:
```suggestion
Many collections have an `entries` method which returns an array of a given object's own enumerable string-keyed property [key, value] pairs.  The `Entries` type will return the type of that collection's entries.
```

I was talking about that you provide (or, give) the `Entries` type a generic parameter of the original collection type for it to return the type.

If you're ok with this wording I'll update in `entry.d.ts` as well.

Reply:
👍

Reply:
ok, updated in https://github.com/sindresorhus/type-fest/pull/130/commits/034a35d68e7bb5c4684867930695cc21eaef4329 and re-requested a review